assets/js/modules/analytics/components/settings/GA4SettingsControls.js

```diff
@@ -211,33 +200,35 @@ export default function GA4SettingsControls( { hasModuleAccess } ) {
 							}
 						/>
 
-						{ enableGA4PropertyTooltip && hasModuleAccess && (
-							<JoyrideTooltip
-								title={ __(
-									'Set up your Google Analytics 4 property here',
-									'google-site-kit'
-								) }
-								target=".googlesitekit-analytics-4__select-property"
-								onDismiss={ () =>
-									setValues( FORM_SETUP, {
-										enableGA4PropertyTooltip: false,
-									} )
-								}
-								cta={
-									<Button
-										className="googlesitekit-tooltip-button"
-										href={ documentationURL }
-										target="_blank"
-										text
-									>
-										{ __(
-											'Learn more',
-											'google-site-kit'
-										) }
-									</Button>
-								}
-							/>
-						) }
+						{ module &&
```

Comment:
Is there a reason we're not using the `isModuleConnected` selector from the `CORE_MODULES` store, passing `analytics-4` as the parameter as described in the IB?

I ask because the goal here is to check if GA4 is not connected. The `module` variable used here is the value of the `getModule` selector from the `CORE_MODULES` store, which always returns a truthy value as long as the respective module is registered.